src/features/content-insights/ContentInsightsSummary.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import ContentInsightsSummaryGhostState from './ContentInsightsSummaryGhostState';
+import GraphCardPreviewsSummary from './GraphCardPreviewsSummary';
+import OpenContentInsightsButton from './OpenContentInsightsButton';
+import { GraphData } from './types';
+
+import './ContentInsightsSummary.scss';
+
+interface Props {
+    graphData: GraphData;
+    isLoading: boolean;
```

Comment:
does this need an isError prop

Reply:
It will eventually, I just want to migrate first and we can do a follow on to add the error experience